app/policies/pageflow/admin/entry_tab_policy.rb

```diff
@@ -0,0 +1,25 @@
+module Pageflow
+  module Admin
+    class EntryTabPolicy
+      attr_reader :user, :tab, :query
+
+      def initialize(user, tab)
+        @user = user
+        @tab = tab
+        @query = EntryRoleQuery.new(user, tab.resource)
+      end
+
+      def see?
```

Comment:
This has also often been the case for my changes. Let us discuss changing the limit